js/lightbeam.js

```diff
@@ -142,7 +142,10 @@ const lightbeam = {
   resetData() {
     const resetData = document.getElementById('reset-data-button');
     resetData.addEventListener('click', async () => {
-      await storeChild.reset();
+      var confirmation = confirm('Pressing OK will delete all'
+       + ' Lightbeam information. Are you sure?');
+      if (confirmation == true)
+        await storeChild.reset();
       window.location.reload();
     });
```

Comment:
@joanasouza thanks for the PR. please enclose the `if` 
```
if (confirmation === true) {
 -- await
 --reload
}
```

Reply:
`confirm` gives true/false values. please make if statement as `if (confirmation)`

Reply:
also, please change `confirmation` to `const`. Thanks :)

Reply:
Thanks, already applied the changes 😄